tests/cmdline/commands/test_calcjob.py

```diff
@@ -258,14 +258,19 @@ def test_calcjob_cleanworkdir(self):
         result = self.cli_runner.invoke(command.calcjob_cleanworkdir, options)
         assert result.exception is None, result.output
 
+        # The flag should have been set
+        assert self.result_job.outputs.remote_folder.base.extras.get('cleaned') is True
+
+        # TODO: This currently fails with sqlite backend!
```

Comment:
Might want to add that is because the filtering relies on the `has_key` filter which does not work properly. Then we know this can be removed once that is fixed.